app/assets/src/components/utils/propTypes.js

```diff
@@ -141,5 +148,6 @@ export default {
   SampleUploadType,
   DownloadType,
   DownloadTypeParam,
+  SampleTypeType,
```

Comment:
I think `SampleType` would be a mistake because of the ambiguity with `Sample`. `SampleTypeType` just seems funny to me, but how about `SampleTypeProps`?